src/components/PageNavLink.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import classNames from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+type Props = {
+  to: string,
+  text: string,
+};
+
+export const PageNavLink: React.FC<Props> = ({ to, text }) => (
```

Comment:
It is recommended to use the default export for the 'pageNavLink' component since it is the primary export of this file.